src/Makefile.stable.toml

```diff
@@ -725,7 +725,7 @@ dependencies = [
 [tasks.ci-coverage-flow]
 description = "Runs the coverage flow and uploads the results to codecov."
 category = "CI"
-condition = { platforms = ["linux"], env = { "TRAVIS" = "true", "CARGO_MAKE_RUN_CODECOV" = "true" } }
+condition = { platforms = ["linux"], env = { "CARGO_MAKE_IS_CI" = "true", "CARGO_MAKE_RUN_CODECOV" = "true" } }
```

Comment:
but wouldn't that trigger for appveyor running linux build? are we sure we can support all CIs on linux?
I'm fine with that assumption if it makes sense.